### DISCUSSION TITLE
在PP-Structure使用文本分析时会输出res.txt,但其中输出得是Unicode字符，如何修改呢

### DISCUSSION THREAD

**Discussion by SolarKnight1** [PaddlePaddle/PaddleOCR#12461] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：2.5.1  PaddleOCR：2.7.0.2 问题相关组件/Related components：
- 运行指令/Command Code：运行了PP-Structure快速开始得第一个代码
- 完整报错/Complete Error Message：{"type": "text", "bbox": [879, 5941, 5056, 7688], "res": [{"text": "\u6ce8\uff1aN\u4ee3\u8868\u672c\u5408\u540c\u7684\u4ea4\u8d39\u671f\u95f4\uff08\u5e74\u5ea6\u6570\uff09", "confidence": 0.9880045652389526, "text_region": [[1105.0, 5954.0], [2911.0, 5958.0], [2911.0, 6054.0], [1105.0, 6050.0]]}, {"text": "\u4e8c\u3001\u5e74\u91d1", "confidence": 0.9993152022361755, "text_region": [[1118.0, 6088.0], [1532.0, 6088.0], [1532.0, 6193.0], [1118.0, 6193.0]]}, {"text": "\u81ea\u672c\u5408\u540c\u751f\u6548\u6ee1\u516d\u4e2a\u4fdd\u5355\u5e74\u5ea6\u540e\u7684\u9996\u4e2a\u5e74\u751f\u6548\u5bf9\u5e94\u65e5\u8d77\u81f3\u672c\u5408\u540c\u4fdd\u9669\u671f\u95f4\u5c4a\u6ee1\u524d\u4e00\u5e74", "confidence": 0.995742678642273, "text_region": [[1123.0, 6227.0], [5039.0, 6227.0], [5039.0, 6327.0], [1123.0, 6327.0]]}, {"text": "\u7684\u5e74\u751f\u6548\u5bf9\u5e94\u65e5\u6b62\uff0c\u82e5\u88ab\u4fdd\u9669\u4eba\u751f\u5b58\u81f3\u672c\u5408\u540c\u7684\u5e74\u751f\u6548\u5bf9\u5e94\u65e5\uff0c\u672c\u516c\u53f8\u6bcf\u5e74\u6309\u4e0b\u5217\u7ea6\u5b9a\u7ed9\u4ed8", "confidence": 0.9989921450614929, "text_region": [[905.0, 6357.0], [5030.0, 6361.0], [5030.0, 6458.0], [905.0, 6453.0]]}, {"text": "\u5e74\u91d1\uff1a", "confidence": 0.999095618724823, "text_region": [[900.0, 6486.0], [1172.0, 6497.0], [1167.0, 6602.0], [895.0, 6591.0]]}, {"text": "\u5e74\u91d1=\u672c\u5408\u540c\u57fa\u672c\u4fdd\u9669\u91d1\u989d\u00d7[1+(\u4fdd\u5355\u7ecf\u8fc7\u6574\u5e74\u5ea6\u6570-6)\u00d75%]\uff0c\u4f46\u6700\u9ad8\u4e0d\u8d85\u8fc7\u672c\u5408\u540c\u57fa\u672c", "confidence": 0.9760488271713257, "text_region": [[1114.0, 6638.0], [5034.0, 6638.0], [5034.0, 6735.0], [1114.0, 6735.0]]}, {"text": "\u4fdd\u9669\u91d1\u989d\u00d7150%", "confidence": 0.9941878914833069, "text_region": [[910.0, 6764.0], [1689.0, 6773.0], [1687.0, 6874.0], [909.0, 6865.0]]}, {"text": "\u4e09\u3001\u5475\u62a4\u91d1", "confidence": 0.9987638592720032, "text_region": [[1111.0, 6898.0], [1637.0, 6907.0], [1635.0, 7012.0], [1109.0, 7003.0]]}, {"text": "\u88ab\u4fdd\u9669\u4eba\u751f\u5b58\u81f3\u672c\u5408\u540c\u4fdd\u9669\u671f\u95f4\u5c4a\u6ee1\u7684\u5e74\u751f\u6548\u5bf9\u5e94\u65e5\uff0c\u672c\u5408\u540c\u7ec8\u6b62\uff0c\u672c\u516c\u53f8\u6309\u672c\u5408\u540c\u57fa", "confidence": 0.9969009757041931, "text_region": [[1110.0, 7041.0], [5034.0, 7046.0], [5034.0, 7142.0], [1110.0, 7138.0]]}, {"text": "\u672c\u4fdd\u9669\u91d1\u989d\u7684150%\u7ed9\u4ed8\u5475\u62a4\u91d1\u3002", "confidence": 0.9963945746421814, "text_region": [[914.0, 7176.0], [2350.0, 7176.0], [2350.0, 7272.0], [914.0, 7272.0]]}, {"text": "\u56db\u3001\u8eab\u6545\u4fdd\u9669\u91d1", "confidence": 0.9939187169075012, "text_region": [[1118.0, 7314.0], [1841.0, 7314.0], [1841.0, 7415.0], [1118.0, 7415.0]]}, {"text": "\u88ab\u4fdd\u9669\u4eba\u4e8e\u672c\u5408\u540c\u751f\u6548\uff08\u6216\u6700\u540e\u590d\u6548\uff09\u4e4b\u65e5\u8d77\u8eab\u6545\uff0c\u672c\u5408\u540c\u7ec8\u6b62\uff0c\u672c\u516c\u53f8\u6309\u88ab\u4fdd\u9669\u4eba\u8eab", "confidence": 0.9964439868927002, "text_region": [[1114.0, 7453.0], [5034.0, 7453.0], [5034.0, 7549.0], [1114.0, 7549.0]]}, {"text": "\u6545\u5f53\u65f6\u4e0b\u5217\u4e24\u8005\u7684\u8f83\u5927\u503c\u7ed9\u4ed8\u8eab\u6545\u4fdd\u9669\u91d1\uff1a", "confidence": 0.9933617115020752, "text_region": [[905.0, 7583.0], [2833.0, 7583.0], [2833.0, 7680.0], [905.0, 7680.0]]}], "img_idx": 0}
{"type": "text", "bbox": [906, 2579, 5053, 5355], "res": [{"text": "\u97e6\u4e00\u4f59", "confidence": 0.536453127861023, "text_region": [[923.0, 2596.0], [1226.0, 2596.0], [1226.0, 2657.0], [923.0, 2657.0]]}, {"text": "\u4fdd\u9669\u5408\u95ee\u6784\u6210", "confidence": 0.8918231129646301, "text_region": [[1186.0, 2588.0], [1951.0, 2579.0], [1952.0, 2665.0], [1187.0, 2675.0]]}, {"text": "\u56fd\u5bff\u946b\u7965\u5b9d\u5e74\u91d1\u4fdd\u9669\u5408\u540c(\u4ee5\u4e0b\u7b80\u79f0\u672c\u5408\u540c)\u7531\u4fdd\u9669\u5355\u53ca\u6240\u9644\u56fd\u5bff\u946b\u7965\u5b9d\u5e74\u91d1\u4fdd\u9669\u5229\u76ca", "confidence": 0.9861629009246826, "text_region": [[1122.0, 2705.0], [5036.0, 2705.0], [5036.0, 2792.0], [1122.0, 2792.0]]}, {"text": "\u6761\u6b3e\uff08\u4ee5\u4e0b\u7b80\u79f0\u672c\u5408\u540c\u5229\u76ca\u6761\u6b3e\uff09\u3001\u4e2a\u4eba\u4fdd\u9669\u57fa\u672c\u6761\u6b3e\uff08\u4ee5\u4e0b\u7b80\u79f0\u672c\u5408\u540c\u57fa\u672c\u6761\u6b3e\uff09\u3001\u73b0\u91d1", "confidence": 0.9971321225166321, "text_region": [[915.0, 2835.0], [5040.0, 2839.0], [5040.0, 2939.0], [915.0, 2935.0]]}, {"text": "\u4ecb\u503c\u8868\u3001\u58f0\u660e\u3001\u6279\u6ce8\u3001\u6279\u5355\u4ee5\u53ca\u4e0e\u672c\u5408\u540c\u6709\u5173\u7684\u6295\u4fdd\u5355\u3001\u590d\u6548\u7533\u8bf7\u4e66\u3001\u5065\u5eb7\u58f0\u660e\u4e66\u548c\u5176\u4ed6", "confidence": 0.9974334836006165, "text_region": [[928.0, 2978.0], [5027.0, 2978.0], [5027.0, 3065.0], [928.0, 3065.0]]}, {"text": "\u4e66\u9762\u534f\u8bae\u5171\u540c\u6784\u6210\u3002", "confidence": 0.9961833953857422, "text_region": [[924.0, 3104.0], [1770.0, 3113.0], [1769.0, 3204.0], [923.0, 3195.0]]}, {"text": "\u7b2c\u4e8c\u6761\u6295\u4fdd\u8303\u56f4", "confidence": 0.9992436766624451, "text_region": [[907.0, 3325.0], [1736.0, 3334.0], [1735.0, 3438.0], [906.0, 3429.0]]}, {"text": "\u51e1\u51fa\u751f\u4e8c\u5341\u516b\u65e5\u4ee5\u4e0a\u3001\u4e03\u5341\u5468\u7f57\u4ee5\u4e0b\uff0c\u8eab\u4f53\u5065\u5eb7\u8005\u5747\u53ef\u4f5c\u4e3a\u88ab\u4fdd\u9669\u4eba\uff0c\u7531\u672c\u4eba\u6216\u5bf9\u5176", "confidence": 0.9641901850700378, "text_region": [[1126.0, 3473.0], [5036.0, 3473.0], [5036.0, 3559.0], [1126.0, 3559.0]]}, {"text": "\u5177\u6709\u4fdd\u9669\u5229\u76ca\u7684\u4eba\u4f5c\u4e3a\u6295\u4fdd\u4eba\u5411\u4e2d\u56fd\u4eba\u5bff\u4fdd\u9669\u80a1\u4efd\u6709\u9650\u516c\u53f8\uff08\u4ee5\u535c\u7b80\u79f0\u672c\u516c\u53f8\uff09\u6295\u4fdd\u672c\u4fdd", "confidence": 0.9907857179641724, "text_region": [[915.0, 3603.0], [5031.0, 3607.0], [5031.0, 3698.0], [915.0, 3694.0]]}, {"text": "\u9669\u3002", "confidence": 0.9700833559036255, "text_region": [[915.0, 3741.0], [1049.0, 3741.0], [1049.0, 3833.0], [915.0, 3833.0]]}, {"text": "\u7b2c\u4e09\u6761\u4fdd\u9669\u671f\u95f4", "confidence": 0.9984227418899536, "text_region": [[919.0, 3967.0], [1735.0, 3967.0], [1735.0, 4058.0], [919.0, 4058.0]]}, {"text": "\u672c\u5408\u540c\u7684\u4fdd\u9669\u671f\u95f4\u5206\u4e3a\u5341\u5e74\u3001\u5341\u4e94\u5e74\u548c\u4e8c\u5341\u5e74\u4e09\u79cd\uff0c\u6295\u4fdd\u4eba\u53ef\u9009\u62e9\u5176\u4e2d\u4e00\u79cd\u4f5c\u4e3a\u672c\u5408\u540c", "confidence": 0.986031174659729, "text_region": [[1113.0, 4097.0], [5036.0, 4097.0], [5036.0, 4197.0], [1113.0, 4197.0]]}, {"text": "\u7684\u4fdd\u9669\u671f\u95f4\u3002", "confidence": 0.9988047480583191, "text_region": [[916.0, 4227.0], [1464.0, 4236.0], [1462.0, 4341.0], [914.0, 4331.0]]}, {"text": "\u7b2c\u56db\u6761\u4fdd\u9669\u8d23\u4efb", "confidence": 0.9629840850830078, "text_region": [[919.0, 4457.0], [1731.0, 4457.0], [1731.0, 4548.0], [919.0, 4548.0]]}, {"text": "\u5728\u672c\u5408\u540c\u4fdd\u9669\u671f\u95f4\u5185\uff0c\u672c\u516c\u53f8\u627f\u62c5\u4ee5\u4e0b\u4fdd\u9669\u8d23\u4efb\uff1a", "confidence": 0.9942008852958679, "text_region": [[1113.0, 4587.0], [3338.0, 4592.0], [3338.0, 4696.0], [1113.0, 4691.0]]}, {"text": "\u4e00\u3001\u7279\u522b\u5173\u7231\u91d1", "confidence": 0.9992284178733826, "text_region": [[1131.0, 4735.0], [1839.0, 4735.0], [1839.0, 4826.0], [1131.0, 4826.0]]}, {"text": "\u5728\u672c\u5408\u540c\u751f\u6548\u6ee1\u4e94\u4e2a\u4fdd\u5355\u5e74\u5ea6\u540e\u7684\u9996\u4e2a\u5e74\u751f\u6548\u5bf9\u5e94\u65e5\uff0c\u82e5\u88ab\u4fdd\u9669\u4eba\u751f\u5b58\uff0c\u672c\u516c\u53f8\u6309\u4e0b\u5217", "confidence": 0.9993001818656921, "text_region": [[1118.0, 4865.0], [5036.0, 4865.0], [5036.0, 4965.0], [1118.0, 4965.0]]}, {"text": "\u7ea6\u5b9a\u7ed9\u4ed8\u7279\u522b\u5173\u7231\u91d1\uff1a", "confidence": 0.9982278943061829, "text_region": [[923.0, 5008.0], [1891.0, 5008.0], [1891.0, 5095.0], [923.0, 5095.0]]}, {"text": "\u7279\u522b\u5173\u7231\u91d1=\u672c\u5408\u540c\u9996\u6b21\u4ea4\u7eb3\u4fdd\u9669\u8d39\uff08\u4e0d\u8ba1\u5229\u606f\uff09\u00d7\u7279\u522b\u5173\u7231\u91d1\u6bd4\u4f8b\uff1b\u5176\u4e2d\uff0c\u7279\u522b\u5173\u7231", "confidence": 0.9954959750175476, "text_region": [[1118.0, 5138.0], [5031.0, 5142.0], [5031.0, 5234.0], [1118.0, 5229.0]]}, {"text": "\u6bd4\u4f2a\u6309\u4e0b\u8868\u89c4\u5b9a\uff1a", "confidence": 0.8378093838691711, "text_region": [[975.0, 5277.0], [1766.0, 5277.0], [1766.0, 5338.0], [975.0, 5338.0]]}], "img_idx": 0}
{"type": "text", "bbox": [904, 2646, 5043, 3220], "res": [{"text": "\u56fd\u5bff\u946b\u7965\u5b9d\u5e74\u91d1\u4fdd\u9669\u5408\u540c(\u4ee5\u4e0b\u7b80\u79f0\u672c\u5408\u540c)\u7531\u4fdd\u9669\u5355\u53ca\u6240\u9644\u56fd\u5bff\u946b\u7965\u5b9d\u5e74\u91d1\u4fdd\u9669\u5229\u76ca", "confidence": 0.9888964295387268, "text_region": [[1115.0, 2695.0], [5026.0, 2700.0], [5026.0, 2790.0], [1115.0, 2785.0]]}, {"text": "\u6761\u6b3e\uff08\u4ee5\u4e0b\u7b80\u79f0\u672c\u5408\u540c\u5229\u76ca\u6761\u6b3e\uff09\u3001\u4e2a\u4eba\u4fdd\u9669\u57fa\u672c\u6761\u6b3e\uff08\u4ee5\u4e0b\u7b80\u79f0\u672c\u5408\u540c\u57fa\u672c\u6761\u6b3e\uff09\u3001\u73b0\u91d1", "confidence": 0.9969751238822937, "text_region": [[917.0, 2839.0], [5030.0, 2839.0], [5030.0, 2929.0], [917.0, 2929.0]]}, {"text": "\u4ecb\u503c\u8868\u3001\u58f0\u660e\u3001\u6279\u6ce8\u3001\u6279\u5355\u4ee5\u53ca\u4e0e\u672c\u5408\u540c\u6709\u5173\u7684\u6295\u4fdd\u5355\u3001\u590d\u6548\u7533\u8bf7\u4e66\u3001\u5065\u5eb7\u58f0\u660e\u4e66\u548c\u5176\u4ed6", "confidence": 0.9899827837944031, "text_region": [[921.0, 2973.0], [5026.0, 2973.0], [5026.0, 3063.0], [921.0, 3063.0]]}, {"text": "\u4e66\u9762\u534f\u8bae\u5171\u540c\u6784\u6210\u3002", "confidence": 0.98348069190979, "text_region": [[922.0, 3094.0], [1767.0, 3108.0], [1765.0, 3203.0], [921.0, 3188.0]]}], "img_idx": 0}
{"type": "text", "bbox": [3488, 948, 5050, 1042], "res": [{"text": "\u56fd\u4eba\u5bff", "confidence": 0.9991995692253113, "text_region": [[3584.0, 957.0], [3892.0, 957.0], [3892.0, 1035.0], [3584.0, 1035.0]]}, {"text": "[2020", "confidence": 0.9037823677062988, "text_region": [[3958.0, 961.0], [4230.0, 961.0], [4230.0, 1035.0], [3958.0, 1035.0]]}, {"text": "\u5e74\u91d1\u4fdd\u9669", "confidence": 0.9998314380645752, "text_region": [[4318.0, 955.0], [4728.0, 958.0], [4727.0, 1035.0], [4318.0, 1032.0]]}, {"text": "\u53f7", "confidence": 0.9931081533432007, "text_region": [[4964.0, 954.0], [5034.0, 954.0], [5034.0, 1033.0], [4964.0, 1033.0]]}, {"text": "216", "confidence": 0.9999452233314514, "text_region": [[4754.0, 964.0], [4925.0, 964.0], [4925.0, 1036.0], [4754.0, 1036.0]]}], "img_idx": 0}
{"type": "text", "bbox": [3477, 1036, 5004, 1843], "res": [{"text": "\u8bf7\u626b\u63cf\u4ee5\u67e5\u8be2\u9a8c\u8bc1\u6761\u6b3e", "confidence": 0.9995121955871582, "text_region": [[3747.0, 1734.0], [4780.0, 1734.0], [4780.0, 1824.0], [3747.0, 1824.0]]}], "img_idx": 0}
{"type": "title", "bbox": [2008, 2194, 3951, 2338], "res": [{"text": "\u56fd\u5bff\u946b\u7965", "confidence": 0.9670926332473755, "text_region": [[2028.0, 2202.0], [2589.0, 2215.0], [2588.0, 2323.0], [2026.0, 2310.0]]}, {"text": "\u76ca\u6761\u6b3e", "confidence": 0.9975270628929138, "text_region": [[3465.0, 2219.0], [3932.0, 2209.0], [3933.0, 2310.0], [3466.0, 2321.0]]}], "img_idx": 0}
{"type": "table", "bbox": [2111, 5357, 3877, 5938], "res": {"cell_bbox": [[14.094593048095703, 17.45352554321289, 662.4326171875, 18.32956314086914, 658.5393676757812, 188.83364868164062, 13.254117965698242, 187.46763610839844], [480.5579833984375, 12.148792266845703, 1759.39892578125, 12.584157943725586, 1759.164306640625, 181.17874145507812, 471.2683410644531, 182.047119140625], [39.68360900878906, 169.44618225097656, 657.8256225585938, 172.888671875, 664.006103515625, 289.7209167480469, 39.746986389160156, 287.38861083984375], [609.55517578125, 167.8386993408203, 1756.1478271484375, 169.1645965576172, 1755.8753662109375, 296.2102355957031, 601.5497436523438, 296.2478942871094], [49.35063552856445, 291.37158203125, 670.898193359375, 293.9410095214844, 677.9568481445312, 406.8261413574219, 49.99809265136719, 405.7102966308594], [649.685546875, 295.52252197265625, 1753.043701171875, 297.3453063964844, 1752.96533203125, 417.53125, 649.585205078125, 416.6674499511719], [44.85356140136719, 412.1197814941406, 663.0820922851562, 412.94830322265625, 670.8812866210938, 554.7883911132812, 45.509952545166016, 554.6348266601562], [705.7468872070312, 415.3524169921875, 1745.156005859375, 414.52996826171875, 1745.0008544921875, 550.3119506835938, 705.326904296875, 550.5305786132812]], "html": "<html><body><table><tr><td>\u4ea4\u8d39\u671f\u95f4</td><td>\u7279\u522b\u5173\u7231\u91d1\u6bd4\u4f8b</td></tr><tr><td>N=3</td><td>60%</td></tr><tr><td>N=5</td><td>100%</td></tr><tr><td>N=10</td><td>100%</td></tr></table></body></html>"}, "img_idx": 0}
{"type": "footer", "bbox": [903, 7846, 2032, 7920], "res": [{"text": "\u56fd\u5bff\u946b\u7965\u5b9d\u5e74\u91d1\u4fdd\u9669\u5229\u76ca\u6761\u6b3e", "confidence": 0.9889557957649231, "text_region": [[909.0, 7856.0], [1744.0, 7856.0], [1744.0, 7911.0], [909.0, 7911.0]]}, {"text": "\uff08\u7b2c\u4e00\u9875\uff09", "confidence": 0.9868079423904419, "text_region": [[1774.0, 7854.0], [2025.0, 7857.0], [2025.0, 7913.0], [1773.0, 7911.0]]}], "img_idx": 0}
{"type": "reference", "bbox": [905, 7848, 1995, 7922], "res": [{"text": "\u56fd\u5bff\u946b\u7965\u5b9d\u5e74\u91d1\u4fdd\u9669\u5229\u76ca\u6761\u6b3e", "confidence": 0.9938564896583557, "text_region": [[912.0, 7860.0], [1744.0, 7860.0], [1744.0, 7910.0], [912.0, 7910.0]]}, {"text": "\uff08\u7b2c\u4e00\u9875", "confidence": 0.9693655371665955, "text_region": [[1753.0, 7858.0], [1988.0, 7858.0], [1988.0, 7913.0], [1753.0, 7913.0]]}], "img_idx": 0}

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by ColorfulDick** [comment]
同样遇到了类似问题，paddleocr==2.7.0.2

**Comment by Liufeiran123** [comment]
问题解决了吗。同样的问题。

**Comment by greatliu** [comment]
我也遇到同样问题。

**Comment by YingxuanW** [comment]
#13746 参考这个回复
